pkg/cmd/admin/router/router.go

```diff
@@ -642,12 +654,12 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 		if cfg.SecretsAsEnv {
 			env.Add(app.Environment{"DEFAULT_CERTIFICATE": string(defaultCert)})
 		} else {
-			// TODO: make --credentials create secrets and bypass service account
 			env.Add(app.Environment{"DEFAULT_CERTIFICATE_PATH": defaultCertificatePath})
 		}
 	}
-
-	secrets, volumes, mounts, err := generateSecretsConfig(cfg, kClient, namespace, defaultCert)
+	env.Add(app.Environment{"DEFAULT_CERTIFICATE_DIR": defaultCertificateDir})
```

Comment:
If we're just refreshing the cert, and an admin can always provide a default certificate via _PATH, why do we need this new env var for _DIR?